pages/about.tsx

```diff
@@ -0,0 +1,74 @@
+import Layout from "@components/layout";
+import { ReactNode } from "react";
+import styles from "styles/pages/About.module.scss";
+
+interface IAbout {
+  title: string;
+  value: string | ReactNode;
+}
+
+const aboutContainers: IAbout[] = [
+  {
+    title: "Team History",
+    value: `We were founded in 2010 (yes, we're ${
+      new Date().getFullYear() - 2010
+    } years old) and we have competed in the FRC Robotics Competition every year since. We operate out of Mounds View High School and the majority of our students attend there as well.`,
+  },
+  {
+    title: "Our Mission",
+    value:
+      "Are main goal is to improve the skills of our team members (Mentors and Students) to help them be more successful in their current and future endeavers in STEM.",
```

Comment:
our vs are

Reply:
I just commented out the entire about page

Reply:
I did fix it too